src/index.md

```diff
@@ -30,3 +63,5 @@ You can run Renovate as:
 - an [Open Source npm package](https://www.npmjs.com/package/renovate)
 - a [pre-built Open Source image on Docker Hub](https://hub.docker.com/repository/docker/renovate/renovate)
 - a [free GitHub App](https://github.com/marketplace/renovate) that is hosted by WhiteSource
+
+[Install our GitHub app now](https://github.com/marketplace/renovate){ .md-button .md-button--primary }
```

Comment:
How about using a button to nudge the reader to install the Renovate app on GitHub? For example:

![install-github-app-button](https://user-images.githubusercontent.com/34918129/164207077-932bd5fa-cdf5-4b4b-98f9-3acd5c43c4be.png)